instrumentation/rack/lib/opentelemetry/instrumentation/rack/middlewares/tracer_middleware.rb

```diff
@@ -82,6 +82,7 @@ def call(env) # rubocop:disable Metrics/MethodLength
                 OpenTelemetry::Instrumentation::Rack.with_span(request_span) do
                   @app.call(env).tap do |status, headers, response|
                     set_attributes_after_request(request_span, status, headers, response)
+                    config[:response_propagators].each { |propagator| propagator.inject(headers) }
```

Comment:
To state the unstated: each propagator should be prepared to grab `Context.current` since we're not explicitly passing context, here. This is indeed the case for the [`ResponseTextMapPropagator`](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/sdk_experimental/lib/opentelemetry/sdk/trace/propagation/trace_context/response_text_map_propagator.rb#L27).